util/secscan/v4/api.py

```diff
@@ -198,7 +224,16 @@ def _join(first, second):
 
         body = {
             "hash": manifest.digest,
-            "layers": [
+            "layers": [],
+        }
+
+        for l in layers:
+            layer_compressed_size = l.layer_info.compressed_size
+            secscan_index_layer_size.observe(layer_compressed_size)
```

Comment:
Do we want to limit on each layer size or the whole manifest size? (sum of all compressed layer sizes)

Reply:
I put a limit on the layer size since that was the limiting factor from Clair's side. The thing I didn't want was not being able to index a manifest that had to 8GB layers when we put a 10GB limit, for example.